airbyte-webapp/src/packages/cloud/views/DefaultView.tsx

```diff
@@ -17,6 +29,11 @@ export const DefaultView: React.FC = () => {
           : `/${RoutePaths.Workspaces}/${workspaces[0].workspaceId}`
       }
       replace
+      // exp-signup-selected-source-definition
+      {...(isSignupSourceSelectorExperiment && {
+        state: { sourceDefinitionId },
```

Comment:
adding `sourceDefinitionId` to the state so we are able to preselect the source after the redirection (`/source/new-source`)